test/cpp/api/rnn.cpp

```diff
@@ -110,15 +110,15 @@ TEST_CASE("rnn") {
     LSTM model(2, 2);
     for (auto& v : model->parameters()) {
       float size = v->numel();
-      auto p = static_cast<float*>(v->data().storage()->data());
+      auto p = static_cast<float*>(v->data().storage()->storage_impl_->data());
```

Comment:
Why not just put the actual methods on Storage? Or even better, implement `->` overloading so you don't have to enumerate all of them.

Reply:
Ditto all the occurrences below.